_docs/operations/updates.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: guide
+title: StorageOS Docs - Update StorageOS version
+anchor: operations
+module: operations/updates
+---
+
+# Update StorageOS version
+
+> Before upgrading, check the 
+> [release notes]({%link _docs/reference/release_notes.md %}) to confirm
+> whether there is a safe upgrade path between versions.
+
+
+StorageOS version upgrades must be planned and executed taking in consideration
```

Comment:
```suggestion
StorageOS version upgrades must be planned and executed taking in consideration
```
s/taking in consideration/taking into consideration/

Reply:
done, thank you.